splice_ck/src/main/java/com/splicemachine/ck/Constants.java

```diff
@@ -69,4 +69,15 @@
 
     public static final String HBASE_CONFIGURATION_ZOOKEEPER_QUORUM = "hbase.zookeeper.quorum";
     public static final String HBASE_CONFIGURATION_ZOOKEEPER_CLIENTPORT = "hbase.zookeeper.property.clientPort";
+
+    public final static String TBL_REGION_COL0 = "index";
```

Comment:
Tabular(SortHint sortHint, String... headers) can accept an array as headers. Maybe better to define column names as 
String TBL_REGION_COLS[] = {"index", "name", "offline", ..} to have it shorter, as constant names for every column not important here.